applications/GeoMechanicsApplication/custom_elements/geo_steady_state_Pw_piping_element.h

```diff
@@ -249,6 +251,9 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoSteadyStatePwPipingElement : publ
         CheckProperty(DENSITY_WATER);
         CheckProperty(DYNAMIC_VISCOSITY);
         CheckProperty(PIPE_HEIGHT);
+        if constexpr (TDim == 3) {
+            CheckProperty(PIPE_WIDTH_FACTOR);
```

Comment:
I'm ( again ) wondering if this is a factor and therefore unitless or a PIPE_WIDTH with length unit. Maybe Jon knows.

Reply:
@mcgicjn2 What is your opinion on this?

Reply:
I asked Jon this morning. They use pipe width factor so keep as is.